super-cool-game/Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -1,72 +1,52 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
 
 public class PlayerMovement : MonoBehaviour {
-    /* Handles the movement of the player.
-     * This script is always attached to the player's character game object,
-     * while PlayerInput is attached only when the character's owner is the local player.
-     */
-    
-    // These ones can be edited in the inspector
-    public float speed = 3.0f;
```

Comment:
4.0f is better than 3.0f

Also is this part a duplicate of the one in PlayerController.cs?

Reply:
It's moved to PlayerController only. These lines are deletions